packages/pirania/files/etc/config/pirania

```diff
@@ -18,5 +18,8 @@ config base_config 'base_config'
     list allowlist_ipv6 'fc00::/7'
     list allowlist_ipv6 'fe80::/64'
     list allowlist_ipv6 '2a00:1508:0a00::/40'
-    list catch_interfaces 'br-lan'
-    list catch_interfaces 'anygw'    
+    option catch_virtual_ifaces 'yes'
```

Comment:
do you really need the "catch_virtual_ifaces" bool? you can just check if the list is empty or not. in fact, the way you wrote your code, this already happens. in bash, the `for interface in $(whatever); do something; done`, if `whatever` returns empty (or whitespace), `something` will not be run. so the sorrounding `if` is unneeded.
and the two bools are unneeded.